test/conftest.py

```diff
@@ -44,11 +44,12 @@ def pytest_configure(config):
 
 @pytest.fixture
 def signed_asset():
-    def _signed_asset(name: str) -> Tuple[bytes, bytes, bytes]:
+    def _signed_asset(name: str) -> Tuple[bytes, bytes, bytes, str]:
         file = _ASSETS / name
         cert = _ASSETS / f"{name}.crt"
         sig = _ASSETS / f"{name}.sig"
+        email = _ASSETS / f"{name}.email"
```

Comment:
oh btw, I didn't include these files since I didn't know if William wanted this email in a text file in the repo (it is in the certs already)